opm/simulators/aquifers/BlackoilAquiferModel.hpp

```diff
@@ -29,11 +29,17 @@
 #include <opm/parser/eclipse/EclipseState/AquiferCT.hpp>
 #include <opm/parser/eclipse/EclipseState/Aquifetp.hpp>
 #include <opm/parser/eclipse/EclipseState/Aquancon.hpp>
-#include <opm/simulators/timestepping/SimulatorTimer.hpp>
+
+#include <opm/output/data/Aquifer.hpp>
+
 #include <opm/simulators/aquifers/AquiferCarterTracy.hpp>
 #include <opm/simulators/aquifers/AquiferFetkovich.hpp>
+#include <opm/simulators/timestepping/SimulatorTimer.hpp>
```

Comment:
I could not see where we need this header file. Please confirm that we need it.

Reply:
> I could not see where we need this header file. Please confirm that we need it.

You're right.  We don't use that header anywhere in this file.  I'll remove it.